specs/agents/log-correlation.md

```diff
@@ -1,7 +1,16 @@
-## Logging Correlation
+## Log correlation
 
 Agents should provide instrumentation/hooks for popular logging libraries in order to decorate structured log records with trace context. In particular, logging that occurs within the context of a transaction should add the fields `trace.id` and `transaction.id`; logging that occurs within a span should add the fields `trace.id`, `span.id`, and optionally `transaction.id`.
 
 By adding trace context to log records, users will be able to move between the APM UI and Logs UI.
 
-See also the [log-onboarding](log-onboarding.md) spec.
+### `enable_log_correlation` configuration
```

Comment:
IIRC, most agents don't have this option and log correlation is just always on by default. Are you suggesting we should add that option to other agents? If so, could we separate it from this spec?

Reply:
You are right, only the Java agent had this (now removed) option, this is confirmed by a quick search on GH : https://github.com/search?q=org%3Aelastic+enable_log_correlation&type=code , so removing it from the spec makes more sense.